src/Tribe/JSON_LD/Order.php

```diff
@@ -150,13 +160,35 @@ public function get_offer( $ticket, $event ) {
 	public function get_ticket_availability( $ticket ) {
 		$stock = $ticket->stock();
 
-		if ( $stock <= 0 && $stock !== '' ) {
+		if ( $stock === 0 ) {
 			return 'SoldOut';
-		}
-		if ( $stock >= 1 && $stock <= $this->low_stock ) {
+		} elseif ( $stock >= 1 && $stock <= $this->low_stock ) {
 			return 'LimitedAvailability';
 		} else {
 			return 'InStock';
 		}
 	}
+
+	/**
+	 * Return the price currency used on the Ticket
+	 *
+	 * @param $ticket
+	 *
+	 * @return mixed
+	 */
+	public function get_price_currency( $ticket ) {
+		$currency = tribe_get_option( 'ticket-commerce-currency-code', 'USD' );
+
+		if ( class_exists( $ticket->provider_class )
```

Comment:
This seems like an overkill.
We should be able to check for `$ticket->provider_class instanceof Tribe__Tickets__Tickets`, and with the change I suggested on the other we will be always safe to call `get_instance()` and `get_currency()`